cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureTemplateBuilder.java

```diff
@@ -43,23 +37,9 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AzureTemplateBuilder.class);
 
-    private static final String GENERAL_PURPOSE = "GeneralPurpose";
-
-    private static final String MEMORY_OPTIMIZED = "MemoryOptimized";
-
-    private static final Set<String> GENERAL_PURPOSE_INSTANCE_TYPES = Set.of("GP_Gen5_2", "GP_Gen5_4", "GP_Gen5_8", "GP_Gen5_16", "GP_Gen5_32");
```

Comment:
What happened to these?

Reply:
Those were extracted to AzureDatabaseTemplateBuilder, they were only used by database.